integr8ly/integreatly-operator
# Multi tenant RHOAM for developer sandbox

# Description

Enable support for multi tenant in RHOAM to allow the operator to be on-boarded on the developer sandbox

# Verification steps

1. Install the operator from this branch using the `multitenant-managed-api` install type `INSTALLATION_TYPE=multitenant-managed-api make cluster/prepare/local && INSTALLATION_TYPE=multitenant-managed-api make code/run`
2. Deploy the dev sandbox operator on the cluster following the steps from [this document](https://docs.google.com/document/d/1TuyJI7n1rMNCsdd7Myx8pycRK_q3p1x20iuVNhxAQrg/edit?usp=sharing)
3. Remove the smtp credentials on the 3scale-smtp secret under the `redhat-rhoam-3scale` namespace
3. Register as a user of the dev sandbox
    1. Go to route menu under the `toolchain-host-operator` namespace
    2. Access the url and login as one of the testing-idp user
    3. Activate the user by running: `oc patch usersignup -p '{"spec":{"states":["approved"]}}' --type=merge <user-sign-id> -n toolchain-host-operator`
4. Check if the user has access to 3scale and SSO under the dashboard section in Openshift 
    

## Type of change
